ptmdmusique/cs464-564-final-project
# feat: install react bootstrap and add navbar with route templates - #1

## Describe your changes

- First PR 🎉 
- Install react-bootstrap
- Add template pages
- Add Navbar with links to those pages and brand logo + name
- Change the page metadata

## Step to test



- Head to the preview URL
- Make sure the navbar is visible
- Make sure there are 4 routes leading to 4 pages and they're clickable
- Make sure it's responsive
- Make sure it shows a hamburger menu in mobile view and it can be open/close

- Also make sure the tab says PokeStats and have the proper favicon
![image](https://github.com/ptmdmusique/cs464-564-final-project/assets/37349324/6812f55c-0d33-4375-9216-1b89f19974d3)


## Issue ticket number and link



- closes #1

## Checklist before requesting a review
